content/en/serverless/installation/java.md

```diff
@@ -26,6 +26,100 @@ aliases:
 Datadog offers many different ways to enable instrumentation for your serverless applications. Choose a method below that best suits your needs. Datadog generally recommends using the Datadog CLI.
 
 {{< tabs >}}
+{{% tab "Datadog CLI" %}}
```

Comment:
the `custom` and `container image` tabs also need to be updated, since datadog-lambda-java is no longer needed

Reply:
Also, is `Set DD_TRACE_ENABLED to true` still needed? Do you know if we default the tracing to be on or not? It's Ok to leave it as is, but the fewer instructions the better.

Reply:
Also under the "custom" section (for both .NET and Java), do you mind helping update the layer ARN section to folllow the pattern from Python? Basically we should show users the right ARN to use GovCloud vs Commercial and arm64 vs x86